addons/web/static/tests/views/fields/properties_field.test.js

```diff
@@ -1396,6 +1398,32 @@ test("properties: name reset", async () => {
     await animationFrame();
     await closePopover();
     expect(".o_property_field:nth-child(2)").toHaveAttribute("property-name", propertyName);
+
+    // Change the definition and check that the name stay the same
+    await click(".o_property_field:nth-child(2) .o_field_property_open_popover");
+    await animationFrame();
+    await click(".o_field_property_definition_kanban input");
```

Comment:
Can't we `await contains(selector).click();` instead of `await animationFrame; await click(selector);`?